dask_geopandas/clip.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import geopandas
+
+from dask.highlevelgraph import HighLevelGraph
+from dask.utils import derived_from
+from dask.base import tokenize
+
+from .core import GeoDataFrame, GeoSeries
+
+
+@derived_from(geopandas.tools)
+def clip(gdf, mask, keep_geom_type=False):
+    if isinstance(mask, GeoDataFrame) or isinstance(mask, GeoSeries):
```

Comment:
```suggestion
    if isinstance(mask, (GeoDataFrame, GeoSeries)):
```

Shorter this way :).